packages/core/src/__tests__/factory.test.ts

```diff
@@ -12,7 +12,7 @@ describe('validation', () => {
   };
 
   const prismInstance = factory<string, string, string, IPrismConfig>(
-    { mock: { dynamic: true }, validateRequest: false, validateResponse: false },
+    { mock: { dynamic: false }, validateRequest: false, validateResponse: false, securityChecks: true },
```

Comment:
just something to think about:
`validateResponse` vs `securityChecks`, both validate and validations are even in the same array, but we use different forms for the flags. What about `securityChecks` -> `checkSecurity` ?

Reply:
Yes, that makes a lot of sense.